controllers/platformnetwork_controller.go

```diff
@@ -12,8 +12,10 @@ import (
 	"github.com/go-logr/logr"
 	"github.com/gophercloud/gophercloud"
 	"github.com/gophercloud/gophercloud/starlingx/inventory/v1/addresspools"
+	hosts "github.com/gophercloud/gophercloud/starlingx/inventory/v1/hosts"
```

Comment:
Need to load diff - large diffs are not rendered by default.
do we have too many changes to accommodate in this PR?

Reply:
you're renaming the import using the same name. just do:
"github.com/gophercloud/gophercloud/starlingx/inventory/v1/hosts" without hosts